pcsx2/GS/Renderers/HW/GSTextureReplacements.cpp

```diff
@@ -212,7 +212,7 @@ std::string GSTextureReplacements::GetGameTextureDirectory()
 	return Path::Combine(EmuFolders::Textures, s_current_serial);
 }
 
-std::string GSTextureReplacements::GetDumpFilename(const TextureName& name, u32 level)
+std::string GSTextureReplacements::GetFilename(const TextureName& name, const std::string& what, u32 level)
```

Comment:
Can you use a better variable name than "what" like "file_name" or something?

Reply:
oops. fixed.

Reply:
Thanks :)